ModuleFile

```diff
@@ -1,7 +1,8 @@
 name 'citops-repose'
-version '1.8.2'
+version '2.0.0-1'
 description "Repose is an api middleware that provides authentication,
 filtering, ratelimitting and several other features, this deploys it."
 project_page 'https://github.com/rackerlabs/puppet-repose'
 
 dependency 'puppetlabs/stdlib', '>= 2.0.0'
+dependency 'camptocamp/systemd', '>= 0.4.0'
```

Comment:
@gregswift Looks like there are some puppet 4 issues in the spec testing. :/ Can we switch back to the justin8 module for the time being with a todo to switch back to camptocamp when its puppet 4 issues are resolved?